src/hooks/useOrders/useOrders.ts

```diff
@@ -48,6 +48,7 @@ export const useOrders = () => {
                 return tx;
             } catch (error) {
                 console.error(error);
+                throw error;
```

Comment:
These errors can happen in placeOrder, placePreOrder, depositOrder, etc.
Do you know if the scope of the ticket is only to fix the placeOrder?

Reply:
`TooManyActiveOrders` is just an example. We should make our app handle all custom errors that are handled in SDK.